tensorboard/plugins/hparams/_keras_test.py

```diff
@@ -165,7 +174,7 @@ def test_reuse_failure(self):
     def test_invalid_writer(self):
         with self.assertRaisesRegex(
             TypeError,
-            "writer must be a `SummaryWriter` or `str`, not None",
+            "writer must be a `SummaryWriter`, `str` or `PathLike`, not None",
```

Comment:
Same comment as above
```suggestion
            "writer must be a `SummaryWriter`, `str`, or `PathLike`, not None"
```